README.md

```diff
@@ -1,54 +1,65 @@
+# docker_ucp
+
 [![Puppet
 Forge](http://img.shields.io/puppetforge/v/puppetlabs/docker_ucp.svg)](https://forge.puppetlabs.com/puppetlabs/docker_ucp)
 [![Build
 Status](https://travis-ci.org/puppetlabs/puppetlabs-docker_ucp.svg?branch=master)](https://travis-ci.org/puppetlabs/puppetlabs-docker_ucp)
 
-1. [Overview](#overview)
-2. [Module Description - What the module does and why it is useful](#module-description)
-3. [Setup - The basics of getting started with docker_ucp](#setup)
-3. [Usage - Configuration options and additional functionality](#setup)
-5. [Limitations - OS compatibility, etc.](#limitations)
-6. [Development - Guide for contributing to the module](#development)
+1.  [Description - What the module does and why it is useful](#description)
+2.  [Setup - The basics of getting started with docker_ucp](#setup)
+    -  [Beginning with docker_ucp](#beginning-with-docker-ucp)
+3.  [Usage - Configuration options and additional functionality](#usage)
+4.  [Limitations - OS compatibility, etc.](#limitations)
+5.  [Development - Guide for contributing to the module](#development)
 
-## Overview
+## Description
 
 The [Docker Universal Control
 Plane](https://www.docker.com/products/docker-universal-control-plane) (UCP)
-module helps with setting up a UCP controller, and joining nodes to it.
-
-## Module Description
+module installs and configures a UCP controller, and joins nodes to it.
 
 This module provides a single class, `docker_ucp`, which uses the
-official `docker/ucp` container to bootstrap a UCP controller, or join
-a node to at existing UCP.
+official [`docker/ucp` container](https://hub.docker.com/r/docker/ucp/) to
+bootstrap a UCP controller or join a node to an existing UCP-managed swarm.
 
 ## Setup
 
-The module assumes Docker is already
-installed on the host. If you would like to do that with Puppet look at
-the [Docker module](https://forge.puppetlabs.com/garethr/docker).
+The module depends on Docker. To install and configure Docker with Puppet, use
+the [`garethr/docker` module](https://forge.puppet.com/garethr/docker).
```

Comment:
Should this be the [`puppetlabs/docker_platform` module](https://forge.puppet.com/puppetlabs/docker_platform) instead?